File: psets/pset0/pset0.py
"""Write a program that does the following in order:
    1. Asks the user to enter a number "x".
    2. Asks the user to enter a numer "y".
    3. Prints out the number "x", raised to the power "y".
    4. Prints out the log (base 2) of "X"."""
import math

x = int(input("Enter number x: "))
y = int(input("Enter number y: "))
print(f"{x}**{y} = {x**y}")
print(f"log({x}) = {math.log(x, 2)}")

File: psets/pset0/pset0.c
#include <stdio.h>
#include <math.h>

int main(void){
    int x;
    int y;

    printf("Enter a number x: ");
    scanf("%d", &x);
    printf("Enter a number y: ");
    scanf("%d", &y);
    printf("%d**%d = %.1f\n", x, y, pow((double)x,(double)y));
    printf("log(%d) = %.1f", x, (log((double)x)/log(2)));
    printf("\n");
    return 0;
    }